iqa_common/executor/executor_kubernetes.py

```diff
@@ -0,0 +1,65 @@
+from iqa_common.executor import Executor, Command, Execution, ExecutionKubernetes
+import os
+
+
+class ExecutorKubernetes(Executor):
+    """
+    Executor that can be used to run Commands in a Pod running on a Kubernetes cluster.
+    This Executor uses the ExecutionKubernetes to run commands through the Kubernetes Client API.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        :param kwargs:
```

Comment:
Parameters are missing type, please provide them. (I assume almost all are str, but to be sure).